x-pack/filebeat/input/cel/input.go

```diff
@@ -1062,6 +1062,7 @@ func newProgram(ctx context.Context, src, root string, vars map[string]string, c
 	opts := []cel.EnvOption{
 		cel.Declarations(decls.NewVar(root, decls.Dyn)),
```

Comment:
Do you want to address the deprecation warnings now?

`cel.VariableDecls(decls.NewVariable(root, types.DynType)),`